Sources/AirwallexRisk/AirwallexRisk.swift

```diff
@@ -1,6 +1,54 @@
-public struct AirwallexRisk {
-    public private(set) var text = "Hello, World!"
+//
+//  AirwallexRisk.swift
+//  AirwallexRisk
+//
+//  Created by Richie Shilton on 5/7/2023.
+//  Copyright © 2023 Airwallex. All rights reserved.
+//
 
-    public init() {
+import Foundation
+
+public class AirwallexRisk {
+    static var shared: AirwallexRisk = .init()
+    private(set) var context: AirwallexRiskContext?
+
+    /// Must be called once as early as possible in the app lifecycle.
```

Comment:
I thought the doc syntax should be something like 
```
/**
*/
```

Reply:
yeah, I will need to clean up all the docs